petab/core.py

```diff
@@ -546,3 +550,48 @@ def get_notnull_columns(df, candidates):
     """
     return [col for col in candidates
             if col in df and not np.all(df[col].isnull())]
+
+
+def create_condition_df(parameter_ids, condition_ids=None):
+    """Create empty condition dataframe
+
+    Arguments:
+        parameter_ids: the columns
+        condition_ids: the rows
+    Returns:
+        An pandas.DataFrame with empty given rows and columns and all nan
+        values
+    """
+
+    data = {'conditionId': []}
+    for p in parameter_ids:
+        data[p] = []
+
+    df = pd.DataFrame(data)
+    df.set_index(['conditionId'], inplace=True)
+
+    if not condition_ids:
+        return df
+
+    for c in condition_ids:
+        df[c] = np.nan
+
+    return df
+
+
+def create_measurement_df():
+    """Create empty measurement dataframe"""
+
+    df = pd.DataFrame(data={
+        'observableId': [],
+        'preequilibrationConditionId': [],
+        'simulationConditionId': [],
+        'measurement': [],
+        'time': [],
+        'observableParameters': [],
+        'noiseParameters': [],
+        'observableTransformation': [],
+        'noiseDistribution': [],
+    })
```

Comment:
Just wondering: It might be good to limit the number of times all column names are defined as much as possible, to avoid mistakes (e.g. misspellings) and simplify updates of the data format.

Reply:
Not sure how we can really do that. Created variables with the column names would work in case of slightly modifying the name, but for all other updates we will have to check everything again anyways.